wolfscript.js

```diff
@@ -158,6 +172,12 @@ window.onload = function () {
     box1.addEventListener("wekbkitAnimationEnd", boxAnimHandler, false);
     box1.addEventListener("MSAnimationEnd", boxAnimHandler, false);
     box1.addEventListener("animationend", boxAnimHandler, false);
+
+    loadWolf(location.hash);
```

Comment:
Call loadWolf inside of the onload body since somebody might try to initially load the URL with a parameter in it.